libraries/chef_ingredient_resource.rb

```diff
@@ -42,6 +42,9 @@ class ChefIngredient < Chef::Resource::LWRPBase
       # Attributes for authenticating to artifactory to download unstable packages
       attribute :artifactory_username, kind_of: String
       attribute :artifactory_password, kind_of: String
+
+      # Attribute to override default omnitruck endpoint
+      attribute :omnitruck_endpoint, kind_of: String
```

Comment:
I think we can remove this. Since we are using omnitruck endpoint for testing only and we can use the attribute if one is configured.